### PR TITLE
rume

### PR DESCRIPTION
**Pull request by Rumemst**

### CODE REVIEW

File: src/GroupStudy/Rume.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package GroupStudy;
+
+import jdk.swing.interop.SwingInterOpUtils;
```
Comment (MobassaraTabassum) [inline src/GroupStudy/Rume.java:3]: please erase it

**Review by MobassaraTabassum — CHANGES_REQUESTED** [review]
please check it again

**Review by Rumemst — COMMENTED** [review]
i already fix it

**Comment by Rumemst** [comment]
done

**Review by Rumemst — COMMENTED** [review]
done solv

**Pull request merged**